File: NowFloats/CX_Products+CoreDataProperties.m
//
//  CX_Products+CoreDataProperties.m
//  NowFloats
//
//  Created by SRINIVASULU on 30/03/17.
//  Copyright © 2017 CX. All rights reserved.
//

#import "CX_Products+CoreDataProperties.h"

@implementation CX_Products (CoreDataProperties)

+ (NSFetchRequest<CX_Products *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"CX_Products"];
}

@dynamic addToCart;
@dynamic categoryType;
@dynamic createdById;
@dynamic favourite;
@dynamic imageUrl;
@dynamic itemCode;
@dynamic json;
@dynamic metaData;
@dynamic name;
@dynamic p3rdCategory;
@dynamic pid;
@dynamic pPrice;
@dynamic pUpdateDate;
@dynamic quantity;
@dynamic storeId;
@dynamic subCategoryType;
@dynamic subCateNameId;
@dynamic type;
@dynamic age;
@dynamic discountprice;

@end

File: NowFloats/CX_Products+CoreDataProperties.swift
//
//  CX_Products+CoreDataProperties.swift
//  NowFloats
//
//  Created by apple on 15/09/16.
//  Copyright © 2016 CX. All rights reserved.
//
//  Choose "Create NSManagedObject Subclass…" from the Core Data editor menu
//  to delete and recreate this implementation file for your updated model.
//

import Foundation
import CoreData

extension CX_Products {

    @NSManaged var addToCart: String?
    @NSManaged var createdById: String?
    @NSManaged var favourite: String?
    @NSManaged var imageUrl: String?
    @NSManaged var itemCode: String?
    @NSManaged var json: String?
    @NSManaged var name: String?
    @NSManaged var pid: String?
    @NSManaged var quantity: String?
    @NSManaged var storeId: String?
    @NSManaged var subCateNameId: String?
    @NSManaged var type: String?
    @NSManaged var pUpdateDate: NSNumber?
    @NSManaged var pPrice: NSNumber?

}
